buildSrc/build.gradle

```diff
@@ -9,18 +9,22 @@ repositories {
 }
 
 dependencies {
-  implementation "com.android.tools.build:gradle:4.1.1"
+  implementation 'com.android.tools.build:gradle:7.0.0-beta05'
```

Comment:
nit: Since AS arctic fox has been released, we can bump this to the stable version. Same with compose 🥳